src/components/ContainerHeader.vue

```diff
@@ -215,6 +220,9 @@ export default {
             let content = TextFormatting.styleBlocksToHtml(blocks, showEmoticons, null);
             return content.html;
         },
+        netUser() {
+            return this.$state.getActiveNetwork().ircClient.user;
```

Comment:
You will need to be more specific about which network you are getting. If in future we have 2 kiwi windows, both with different ContainerHeader instances, the `getActiveNetwork()` will be incorrect for one of them. You have the `buffer` instance here already, so `buffer.getNetwork()` will always return the correct network